collectRatesData.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+
+const config = require('./config');
+const helpers = require('./helpers');
+
+const logger = helpers.logger
+
+const runLogic = () => {
+    logger.info('✋ Init the collect rates data')
+
+    helpers.writeDataFileFromJson(config.rates)
+
+    logger.info('🔆 Starting to collect rates data')
+
+    const collectWorker = async () => {
+        const dataAsJson = helpers.readDataFileAsJson()
+
+        for(let iOfDataAsJson in dataAsJson) {
```

Comment:
Any good reason you used this `for` and not `forEach`?
There might be a good reason (really) but you need to know why, you might need to explain it

Reply:
Ok that a good reason, but there's a better/another solution, let's talk about it